app/Services/ExchangeRateGetter.php

```diff
@@ -4,68 +4,93 @@
 class ExchangeRateGetter {
 	public function get() {
 		
-	/// Receive JSON exchange rates
+		/// RECEIVE JSON TODAY'S EXCHANGE RATES
 		$arrContextOptions = [
             "ssl" => [
                 "verify_peer" => false,
                 "verify_peer_name" => false,
             ],
         ];
 
-    $path = file_get_contents(
-    	"https://www.cbr-xml-daily.ru/daily_json.js", 
-    	false, 
-    	stream_context_create($arrContextOptions)
-    );
+	    $path = file_get_contents(
+	    	"https://www.cbr-xml-daily.ru/daily_json.js", 
```

Comment:
Create file `exchange-rates.php` and add associative array in this file, add key `daily_rates_url`  and value `"https://www.cbr-xml-daily.ru/daily_json.js"` then place it in config folder

Instead of writing url name directly in anywhere use `config('exchange-rates.daily_rates_url')`